__tests__/app.test.js

```diff
@@ -162,6 +162,97 @@ describe("news-api", () => {
             })
         })  
     })
+    describe("POST requests", () => {
+        describe("POST /api/articles/:article_id/comments", () => {
+            test("responds with status code 201 and the posted object in expected format", () => {
+                return request(app)
+                .post("/api/articles/1/comments")
+                .send({"username": "butter_bridge", "body": "test"})
+                .expect(201)
+                .then(({body}) => {
+                    const { comment } = body
+                    expect(comment.comment_id).toEqual(expect.any(Number))
+                    expect(comment.votes).toEqual(expect.any(Number))
+                    expect(comment.created_at).toEqual(expect.any(String))
+                    expect(comment.author).toEqual(expect.any(String))
+                    expect(comment.body).toEqual(expect.any(String))
+                    expect(comment.article_id).toEqual(expect.any(Number))
```

Comment:
In this case we can assert directly what article_id, author and body is, because these are all given to us, in the url, as well as the new comment which is given to us.